objects.js

```diff
@@ -2640,87 +2640,135 @@ SpriteMorph.prototype.reporterize = function (expressionString) {
     // highly experimental Christmas Easter Egg 2016 :-)
     var ast;
 
-    function parseInfix(expression, operator, already) {
-        // very basic binary infix parser for arithmetic expressions
```

Comment:
This comment should be retained, though in an updated form, I think - including what happens when an error is thrown.